embedded/templates/pages/resume.gohtml

```diff
@@ -0,0 +1,10 @@
+{{define "content"}}
+<p>As this is a public resume, I have purposefully redacted my address and phone number.</p>
+<iframe
+    src="https://drive.google.com/viewerng/viewer?embedded=true&url=https://trevor-edris.io#toolbar=0&scrollbar=0"
```

Comment:
This url will need to change to `https://trevor-edris.io/downloadresume` once this API is hosted at the domain.